### PR TITLE
feat: bumping echarts plugin, adding new treemap plugin

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

Bumps the Echarts plugin, and replaces the old treemap with the new one. We need to consider a proper migration, but this PR and its ephemeral environment can serve as a preview for both the new treemap, and for the updated/polished funnel.

@junlincc

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

After:
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by junlincc** [comment]
because we didn't give the new chart a new name, looks like all the old treemap already migrated to echarts treemap? that will be great we dont even need migration script. 😅

File: superset-frontend/src/visualizations/presets/MainPreset.js [hunk 1]
```diff
@@ -122,7 +123,6 @@ export default class MainPreset extends Preset {
         new TableChartPlugin().configure({ key: 'table' }),
         new TimePivotChartPlugin().configure({ key: 'time_pivot' }),
         new TimeTableChartPlugin().configure({ key: 'time_table' }),
-        new TreemapChartPlugin().configure({ key: 'treemap' }),
```
Comment (zhaoyongjie) [inline superset-frontend/src/visualizations/presets/MainPreset.js:125]: Maybe not a direct replacement from nvd3-treemap to echart -treemap
1) We should need some migration scripts,  adapt to the new Treemap.
2) fix original cypress CI

**Comment by junlincc** [comment]
@rusackas can you push a commit to rename the echarts treemap to treemap 2 to get the ci pass? my understanding is that we still need to work on the migration script....

**Comment by zhaoyongjie** [comment]
- bumping @superset-ui/plugin-chart-echarts 0.17.47
- rename echart treemap to treemap v2
- keep the legacy treemap
- fix treemap issues by https://github.com/apache-superset/superset-ui/pull/1103

thanks @stephenLYZ  for your hard works!

**Comment by junlincc** [comment]
@rusackas @zhaoyongjie can we get the checks pass? thank you for bumping the plugins!

File: superset-frontend/package.json [hunk 2]
```diff
@@ -91,7 +91,7 @@
     "@superset-ui/legacy-preset-chart-big-number": "^0.17.42",
     "@superset-ui/legacy-preset-chart-deckgl": "^0.4.6",
     "@superset-ui/legacy-preset-chart-nvd3": "^0.17.42",
-    "@superset-ui/plugin-chart-echarts": "^0.17.44",
+    "@superset-ui/plugin-chart-echarts": "^0.17.47",
```
Comment (villebro) [inline superset-frontend/package.json:94]: I think bumping this to `0.17.47` without also bumping `superset-ui/core` first to `0.17.46` might be problematic. I suggest merging this first: #14547 , after which this should update cleanly.

**Review by villebro — APPROVED** [review]
Code LGTM

**Pull request merged**